client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/types.ts

```diff
@@ -4,7 +4,7 @@ export interface Task {
 	disabled: boolean;
 	title?: string;
 	subtitle?: string;
-	badgeText?: string;
+	badge_text?: string;
 	actionDispatch?: () => void;
 	isLaunchTask?: boolean;
```

Comment:
I mentioned above we probably want to define isLaunchTask on the backend. If we do, we'll probably be using is_launch_task, and we'd need to update the type here. I think we can/should defer this to a follow up.

Reply:
`isLaunchTask` is defined on the backend. Following the naming php convention can be added in a follow-up PR.